spec/requests/landing_pages_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe "LandingPages", type: :request do
+  describe "GET /landing_pages" do
+    before do
+      visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.